opentargets.json

```diff
@@ -1449,9 +1449,10 @@
     "variantId": {
       "type": "string",
       "description": "Identifier of the variant using CHROM_POS_REF_ALT notation",
-      "pattern": "^[0-9XY]{1,2}_\\d+_[ACTG]+_[ACTG]+$",
+      "pattern": "^[0-9XY]{1,2}|MT_\\d+_[A-Z]+_[A-Z]+$",
```

Comment:
The OR clause should be within the same group like this:

`(^[0-9XY]{1,2}|MT)_\\d+_[A-Z]+_[A-Z]+$`

Reply:
It's now fixed. The newly updated EVA evidence validates (`gs://otar012-eva/21.04/cttv012-2021-03-06.json.gz`) without any issues.